pkg/server/container_attach.go

```diff
@@ -29,13 +29,6 @@ import (
 
 // Attach prepares a streaming endpoint to attach to a running container, and returns the address.
 func (c *criContainerdService) Attach(ctx context.Context, r *runtime.AttachRequest) (retRes *runtime.AttachResponse, retErr error) {
```

Comment:
nit:  The name of argument may be removed.  e.g. retRes, retErr

Reply:
Will do

Reply:
Done.